torchvision/prototype/transforms/_geometry.py

```diff
@@ -272,42 +273,31 @@ def apply_recursively(obj: Any) -> Any:
         return apply_recursively(inputs if len(inputs) > 1 else inputs[0])
 
 
-class RandomZoomOut(Transform):
+class Pad(Transform):
     def __init__(
-        self, fill: Union[float, Sequence[float]] = 0.0, side_range: Tuple[float, float] = (1.0, 4.0), p: float = 0.5
+        self,
+        padding: Union[int, Sequence[int]],
+        fill: Union[float, Sequence[float]] = 0.0,
+        padding_mode: str = "constant",
```

Comment:
Nit: What happened to `Literal`? :)

Reply:
I forgot about it :slightly_smiling_face: